Resources/doc/1-setting_up_the_bundle.rst

```diff
@@ -45,7 +45,7 @@ the following (in the given order) to determine the serializer to use:
 
 #. The one you configured using ``fos_rest.service.serializer`` (if you did).
 #. The JMS serializer, if the `JMSSerializerBundle`_ is available (and registered).
-#. The `Symfony Serializer`_ if it's enabled (or any service called ``serializer``).
+#. The `Symfony Serializer`_ if it's enabled (or any service called ``fos_rest.serializer.symfony``).
```

Comment:
I would just remove this comment, this is an advanced feature that's useless to most users and this comment doesn't truly help since that's not the only condition.